pages/select/select.test.page.tsx

```diff
@@ -70,87 +71,89 @@ export default function SelectPage() {
 
   return (
     <article>
-      <Box padding="l">
-        <Box padding="s">
-          <Box variant="h1">Overflow test</Box>
-          <Box variant="p">Dropdown should not be rendered beyond the container with overflow hidden</Box>
-          <div id="smallest_container" style={{ overflow: 'hidden', height: '500px', padding: 0 }}>
-            <div style={{ overflow: 'hidden', height: '900px' }}>
-              <Select
-                id="select_overflow"
-                placeholder="Choose option"
-                selectedOption={null}
-                options={optionsOverflow}
-              />
+      <ScreenshotArea>
```

Comment:
Why did you decide to convert this page into screenshot testing, when we already have another dedicated one: https://github.com/cloudscape-design/components/blob/main/pages/select/screenshot.page.tsx

Reply:
to avoid the risk of adding a completely new scenario to an existing page (as none of the selects on that page have enough items to be suitable for this test) impacting the correctness & reliability of those existing tests